app.js

```diff
@@ -0,0 +1,14 @@
+const close = document.querySelector('.close');
+const menuBars = document.querySelector('.menu-bars');
+const menu = document.querySelector('#mobile-nav');
+
+function showMobileMenu() {
+  menu.style.display = 'block';
+}
+
+function hideMobileMenu() {
+  // menu.style.display = 'none';
```

Comment:
- [ ] Please kindly uncomment the following lines `menu.style.display = 'none';`.  Currently, when the user clicks the close (X) button, the mobile menu does not disappear. This is because of the commented lines inside `hideMobileMenu()` function. 💬